backend/arm64/proc.ml

```diff
@@ -129,12 +126,13 @@ let register_name ty r =
     (* CR mslater: (SIMD) arm64 *)
     fatal_error "arm64: got vec128 register"
   | Float32 ->
-    (* CR mslater: (float32) arm64 *)
-    fatal_error "arm64: got float32 register"
+    float32_reg_name.(r - first_available_register.(1))
   | Valx2 ->
     (* CR mslater: (SIMD) arm64 *)
     fatal_error "arm64: got valx2 register"
 
+(* CR gyorsh for xclerc: [rotate_registers] used in [coloring] on Mach,
+   but not in IRC on CFG. Are we dropping an optimization here? *)
 let rotate_registers = true
```

Comment:
Possibly; I would remove the binding,
and create an issue to be sure we do
something about it.